tests/cases/fourslash/formattingJsxTexts2.ts

```diff
@@ -0,0 +1,79 @@
+/// <reference path='fourslash.ts' />
+
+//@Filename: file.tsx
+//// const a = (
+////     <div>
+////   foo
+////           </div>
+//// );
+//// 
+//// const b = (
+////     <div>
+////   {     foo  }

```

Comment:
Huh, how did you fix this? It seems like the changes are only related to JsxText, but these spaces fall within a JsxExpression.

Reply:
This got fixed because I have changed how we rescan jsx text and parenthesis. I believe in the failure case, it didn't actually identify any rescan the JsxText, this leaving the code unchanged. Now JsxText is rescaned allowing the JsxExpression to be identified and formatted correctly.

Reply:
Ah, I didn’t see there is actually a rule for `NoSpaceAfterBraceInJsxExpression`